tests/time_tests/c++/pipeline.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include "timer.h"
+
+#include <inference_engine.hpp>
+using namespace InferenceEngine;
+
+
+int runPipeline(const std::string &model, const std::string &device) {
+    start_timer("first_time_inference");
```

Comment:
Start time can return an object which stops the timer on destroy. This also enforces correct creation order if we want to measure "Core ie" construction. 
Please take a look at ScopedTask and OV_ITT_SCOPED_TASK design at openvino\itt\include\openvino\itt.hpp. It worth reusing those ideas.

Reply:
Done

Reply:
Done via SCOPED_TIMER and specification of scopes